.github/workflows/apply.yml

```diff
@@ -7,48 +7,54 @@ on:
         description: The environment to deploy against.
         type: string
         default: intg
+      project:
+        required: true
+        description: The project to run this for
+        type: choice
+        options:
+          - tdr
+          - dr2
 permissions:
   id-token: write
   contents: write
 jobs:
+  setup:
+    uses: nationalarchives/dr2-github-actions/.github/workflows/set_terraform_var_names.yml@main
+    with:
+      project: ${{ github.event.inputs.project }}
+      environment: ${{ github.event.inputs.environment }}
   apply:
-    uses: nationalarchives/tdr-github-actions/.github/workflows/terraform_apply.yml@main
+    uses: nationalarchives/dr2-github-actions/.github/workflows/terraform_apply.yml@main
```

Comment:
This is referencing a dr2 repo, does there need to be something to point to the tdr equivalent repo as well? If the repo is a project neutral one, should it be given the name `da-github-actions` instead?

Reply:
I did originally have it as `da-github-actions` but it was really difficult to get it generic enough for everyone so I made a dr2 specific one. But then the `da-terraform-configurations` repo came along and let me create a generic terraform job. So this repo is now partially generic deploy jobs and partially dr2 specific jobs so I don't really want to change it now. If I can think of a way of making the other workflows generic I will. 
I didn't use the tdr one because changing that would break a lot of jobs whereas this repo isn't being used by much.

Reply:
Maybe put something in the `README` and/or a comment in the code about this. It looks a bit odd given this is a "tdr-" repo, so some explanation is worthwhile.

Reply:
I've added in a comment to the README.